pkg/apis/cluster/v1alpha1/cluster_types.go

```diff
@@ -32,6 +32,9 @@ type ClusterSpec struct {
 
 	// Connection holds info to connect to the member cluster
 	Connection Connection `json:"connection,omitempty"`
+
+	// ExternalKubeAPIEnabled export kubeapiserver to public use a lb type service if connection type is proxy
+	ExternalKubeAPIEnabled bool `json:"externalKubeAPIEnabled,omitempty"`
```

Comment:
It's better to manage the resource define in one place.

Reply:
I'll create another pr to change the way of API import

Reply:
Could we define this using annotation instead? For example, using `tower.kubesphere.io/use-lb-service: true`,  since the field is only used when the cluster connection type is proxy, it's a little bit confusing to declare it as a field.